### PR TITLE
Avatar component designed

### PR DESCRIPTION
**Pull request by SAYUK09**
Good Description.

### CODE REVIEW

**Comment by arjun-computer-geek** [comment]
Hey it's ne Arjun

File: index.html [hunk 1]
```diff
@@ -8,5 +8,7 @@
   </head>
   <body>
     <h1>NEOG CAMP</h1>
+
+    <div>AVATAR COMPONENET</div>
```
Comment (thesudeshdas) [inline index.html]: Looks great..

**Review by thesudeshdas — CHANGES_REQUESTED** [review]
Needs better semantic HTML

File: index.html (hunk 1 above)
Comment (iRohitGaur) [inline index.html]: fix the spelling of component

**Review by MayankKumar10 — COMMENTED** [review]
Looks good

**Comment by Developers-codz** [comment]
awesome

File: index.html [hunk 2]
```diff
@@ -8,5 +8,7 @@
   </head>
   <body>
     <h1>NEOG CAMP</h1>
+
```
Comment (zoef-yt) [inline index.html:11]: The extra space can be removed

**Review by thesudeshdas — APPROVED** [review]
Hey, this is great. You can go ahead and merge it

**Comment by omkarpatke** [comment]
Hello, This is Omkar.

**Review by zoef-yt — CHANGES_REQUESTED** [review]
extra space can be removed

File: index.html (hunk 2 above)
Comment (nayyyhaa) [inline index.html:11]: Looks good

**Review by nayyyhaa — APPROVED** [review]
Changes looks good.

**Review by iRohitGaur — APPROVED** [review]
looks good

File: index.html (hunk 1 above)
Comment (SAYUK09) [reply index.html]: cool. I'll do it.

**Review by manojsarna — APPROVED** [review]
now its fine.

File: index.html (hunk 2 above)
Comment (SAYUK09) [reply index.html:11]: But I think, It will help in readability

**Comment by 05him** [comment]
Let's Hack NASA with this

File: index.html (hunk 2 above)
Comment (Shivani-1524) [inline index.html:11]: looks good !

**Pull request merged**